src/main/java/org/opensearch/ad/model/AnomalyDetector.java

```diff
@@ -72,7 +72,7 @@ public class AnomalyDetector implements Writeable, ToXContentObject {
         it -> parse(it)
     );
     public static final String NO_ID = "";
-    public static final String ANOMALY_DETECTORS_INDEX = "opendistro-anomaly-detectors";
+    public static final String ANOMALY_DETECTORS_INDEX = ".opendistro-anomaly-detectors";
```

Comment:
Just curious, why was the period removed for this system index name?

Reply:
Good question. Here is the [ref](https://github.com/opensearch-project/opensearch-sdk-java/issues/241#issuecomment-1310693864)